include/network/uri/detail/encode.hpp

```diff
@@ -110,7 +110,7 @@ OutputIterator encode_query(InputIterator first, InputIterator last,
                             OutputIterator out) {
```

Comment:
I think `encode_query` should be removed altogether. Neither the whole query part of an URI nor a single query key/value pair should be encoded at once, because of the ambiguity problem of separators (consider a query like `a=b=c`, which `=` is the separator and which should be percent-encoded?).